app/controllers/forms_controller.rb

```diff
@@ -223,52 +225,52 @@ def remove_questions
   # makes an unpublished copy of the form that can be edited without affecting the original
   def clone
     begin
-      cloned = @form.replicate(:mode => :clone)
+      cloned = @form.replicate(mode: clone)
 
       # save the cloned obj id so that it will flash
       flash[:modified_obj_id] = cloned.id
 
-      flash[:success] = t("form.clone_success", :form_name => @form.name)
+      flash[:success] = t("form.clone_success", form_name: @form.name)
     rescue
-      flash[:error] = t("form.clone_error", :msg => $!.to_s)
+      flash[:error] = t("form.clone_error", msg: $!.to_s)
```

Comment:
Not sure what to do with this

Reply:
Best is to do `rescue Exception => e` then you can do `e.to_s` instead of `$!`. If you can be more specific about what you're catching, even better. But `Exception` works as a drop-in replacement.